src/modules/orders/infrastructure/repositories/avvcatalogue.repository.ts

```diff
@@ -1,10 +1,12 @@
-import { AVVCatalog } from '../model/avvcatalog.entity';
+import { AbstractRepository } from '../../../shared/infrastructure';
+import { AVVCatalogueObject } from '../../../shared/infrastructure/parse-types';
+import { AVVCatalog } from '../../legacy/model/avvcatalog.entity';
 import {
     AVVCatalogData,
     MibiCatalogData,
     MibiCatalogFacettenData,
     AVVCatalogObject
-} from '../model/legacy.model';
+} from '../../legacy/model/legacy.model';
```

Comment:
The non-legacy code should not have any dependencies on the legacy code.  The other way round is OK.  If need be we need to take the required Objects out of the legacy code, but then they should be proper Entities or Value Objects.

Reply:
refactored in #97